packages/react-native-reanimated/src/logger/logger.ts

```diff
@@ -36,7 +40,7 @@ function logToConsole(data: LogData) {
 export const DEFAULT_LOGGER_CONFIG: LoggerConfigInternal = {
   logFunction: logToConsole,
   level: LogLevel.warn,
-  strict: false,
+  strict: true,
```

Comment:
I think that we'd better enable strict mode by default and tell users how to disable it if warnings are too annoying.